File: 01intro/Variables.java

public class Variables
{
  public static void main(String[] args)
  {
    byte a;
	short b;
	int c;
	long d;

	d = c = b = a = 14;

	float y;
	double z;

	y = a / 3.0f;
	z = y*5;

	d = (long)z;
	System.out.println("d = " + d);
	System.out.println("z = " + z);

	char i;
	i = 'i';

	boolean j;
	j = y < z;
	System.out.println("y < z " + i + "s " + j);
  }
}

File: 01intro/variables.cc
#include <iostream>
  using namespace std;

int main(int args, char** argv) {
  unsigned char a;
  short b;
  int c;
  long d;

  d = c = b = a = 14;

  float y;
  double z;

  y = a / 3.0f;
  z = y*5;

  d = static_cast<long>(z);
  cout << "d = " d << "\n";
  cout << "z = " << z << endl;

  char i;
  i = 'i';

  bool j;
  j = y < z;
  cout << "y < z " << i << "s " << j << endl;
}

